EcoSoapBankTests/ColorTests.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ColorTests.swift
+//  EcoSoapBankTests
+//
+//  Created by Shawn Gee on 8/10/20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+
+import XCTest
+@testable import EcoSoapBank
+
+class ColorTests: XCTestCase {
+    
+    // swiftlint:disable empty_xctest_method
+    func testColorsInitializeProperly() {
+        let _: UIColor = .esbGreen
+        let _: UIColor = .downyBlue
+        let _: UIColor = .codGrey
+        let _: UIColor = .montanaGrey
+        let _: UIColor = .shuttleGrey
+        let _: UIColor = .silver
+    }
```

Comment:
This works fine, though I wonder if it'd be more "best practice" to use `XCTAssertNonNil`? (I don't think it makes a meaningful difference but the thought occurred.)

Reply:
I can try that. The problem is I am force unwrapping the colors anyway with these static properties so that we don't have to deal with optionals, so not sure if it will let me assert non nil for a non-optional.

Reply:
Turns out you can use XCTAssertNonNil with non-optional parameters.